tests/o11y/o11y-deployments.go

```diff
@@ -0,0 +1,100 @@
+package o11y
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+)
+
+var _ = framework.O11ySuiteDescribe("O11Y E2E tests for Deployments", Label("o11y", "HACBS"), Pending, func() {
```

Comment:
Is it supposed to be "Pending"? Same for `o11y-pipelinerun.go`

Reply:
yes. Deployment is WIP. SO it should be in pending